packages/ui-module-header/src/components/headerSearch.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Input } from '@test-assessment/ui-components';
+import { HeaderSearchProps } from '../types';
+
+export const HeaderSearch = (props: HeaderSearchProps) => {
+  const {
+    title,
+    searchPlaceholder,
+    score,
+    primaryButton,
+    secondaryButton,
+    thirdButton,
+  } = props;
+  const styles = useStyles();
+
+  return (
+    <div className={`${styles.root}`}>
+      <div className={`${styles.title}`}>{title}</div>
+      <div className={`${styles.searchWrap}`}>
+        <Input type="search" placeholder={searchPlaceholder} />
```

Comment:
@vunam2000 Could you add an event onChange to check the change& enter pressed (just a placeholder) on the searchbox ?